modules/guardduty-org-admin-account/README.md

```diff
@@ -0,0 +1,45 @@
+# terraform-aws-tardigrade-guardduty/org_admin_account
```

Comment:
Rename directory to `organization-admin`

```suggestion
# terraform-aws-tardigrade-guardduty/organization-admin
```

Reply:
Updated